added FAQ link to home page, updated all info on FAQ page to reflect …

Eyetsko opened the pull request:
Added FAQ page to TP

## What Github issue does this PR relate to? Insert link.
FAQ for the Talent Pool #793
https://github.com/talent-connect/connect/issues/793

## What should the reviewer know?

added FAQ link to home page
added to route
added files for pages/front under talent pool
updated all info on FAQ page to reflect TP questions

See Screenshots below:

![TP - FAQ - 1](https://github.com/talent-connect/connect/assets/104879063/1d039687-69be-44c9-8818-eed03a8f66c1)
![TP - FAQ - 2](https://github.com/talent-connect/connect/assets/104879063/3e693346-0263-460e-b98a-143969a08cc5)
![TP - FAQ - 3](https://github.com/talent-connect/connect/assets/104879063/551480c2-7aaa-4e8a-aa2c-24b2aea8431e)
![TP - HOME - FAQ LINK](https://github.com/talent-connect/connect/assets/104879063/170d08d1-c536-4b32-9706-682e6d006d77)

astkhikatredi reviewed (approved):
Hi @Eyetsko, thanks a lot for the wonderful work, looks good for me.
Just one question: did you use the same font as on CON? It seems to my eyes that the font is different, but maybe this is a problem with my eyes.

thanks

astkhikatredi commented:
@Eyetsko Emily, one more question, could you please help as as native English speaker?
What is better to use in this case: Guidance or Guidlines?

astkhikatredi commented:
Thanks dear @Eyetsko.! 
RE: font question: I will let the guys to reply you.
RE: guidance/guidelines: do you think you can change from Guidelines to Guidance in FAQ on CON? Do you need a ticket for that?
Thanks

> @astkhikatredi
> 
> * RE: font - I had a similar thought but there is actually no css in the file for connect either, so I'm actually not sure where it is inheriting the font from. Maybe someone else can advise?
> * RE: guidance/guidelines - I think Guidelines is more common but I actually think Guidance sounds gentler/kinder

Eyetsko commented:
@astkhikatredi - I created a new pull request with the changes requested (changed guidelines to guidance in conn FAQ) - however, this version does not include TP FAQ changes as I was unable to pull that branch back down.

katamatata commented on apps/redi-talent-pool/src/pages/front/Faqs.scss:
I guess we can remove this file since it's empty.

katamatata commented on apps/redi-talent-pool/src/assets/locales/en/translation.json:
```diff
@@ -51,127 +55,122 @@
-            "question": "How can we structure our mentoring sessions?",
-            "answer": "For your first meeting you can focus on getting to know each other and work on setting goals together. Some students already have clear long-term goals. In these cases, you can think carefully about concrete, realistic goals for the next 6 months that bring you closer to your long-term goals. Other students are not sure yet about how to move forward, in which area to specia ...
+            "question": "Are there any requirements for accessing the ReDI Talent Pool as a company?",
+            "answer": "The ReDI Talent Pool serves as a safe space for the ReDI Community and therefore it is also not open to everybody. To access the ReDI Talent Pool as a company representative your company must be a ReDI Career Partner. You should get in touch with our partnership team (partner@redi-school.org) in order to find out how to become a ReDI Career Partner."
```
We can make partner@redi-school.org a link here. Just like you do on the line 129.

Eyetsko commented:
@katamatata 
- removed faqs.scss file because it was empty (however, I did not delete the same file under connect even though it is also empty because I did not create that file)
- changed email address to clickable link

ericbolikowski reviewed (approved):
Looks good!
